### ISSUE TITLE
Filter by multiple tournament types

### ISSUE THREAD

**Issue by bluerook17** [haslo/lists_juggle_browser#59]
I'm not sure if this is already done, but can we filter by multiple tournament types? For example, filtering by Regional or higher-level events.

**Comment by haslo** [comment]
Good idea. Currently not possible, but it wouldn't be that hard to implement. I'll look into it 👍

**Comment by haslo** [comment]
I've got a proof of concept for a multi-select, but I'm not happy with its usability. I think I'll instead add some hard-coded entries that include multiple types to the dropdown.
